Aspose.Slides.WebExtensions/Templates/common/bullet.html

```diff
@@ -18,12 +18,13 @@
     }
     else
     {
-        margins[2] -= 12;  // FIXME magic number (ideally it should be bullet width)
+        margins[2] = -4;

```

Comment:
@polonair ,

Meaningful comment about magic number disappear, but -4 not looks less magic then -12.

Reply:
[remove magic constant, just clean spaces between spans](https://github.com/aspose-slides/Aspose.Slides.WebExtensions/pull/7/commits/d70bed1aaad8517743008342951aea11c62d77e6)